modules/oc-compliance-using-scan-setting-bindings.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * security/oc_compliance_plug_in/oc-compliance-plug-in-using.adoc
+
+[id="using-scan-setting-bindings_{context}"]
+= Using ScanSettingBinding custom resources
+
+When using the `ScanSetting` and `ScanSettingBinding` custom resources (CRs) that the Compliance Operator provides, it is possible to run scans for multiple profiles while using a common set of scan options, such as `schedule`, `machine roles`, `tolerations`, and so on. While that is easier than working with multiple `ComplianceSuite` or `ComplianceScan` objects, it can confuse new users.
+
+The `oc-compliance` `bind` subcommand helps you create a `ScanSettingBinding` CR.
+
+.Procedure
+
+. Run:
++
+[source,terminal]
+----
+$ oc compliance bind [--dry-run] -N <binding name> [-S <scansetting name>] <objtype/objname> [..<objtype/objname>]
+----
++
+* If you omit the `-S` flag, the `default` scan setting provided by the Compliance Operator is used.
+* The object type can be `profile` or `tailoredprofile`. More than one object can be provided.
+* Add the `--dry-run` option to display the YAML file of the objects that are created.
```

Comment:
Add `objtype` & `objname` explanation for above command

Reply:
@JAORMX Can you help add details on the explanation?

Reply:
object type is the kubernetes object type (e.g. Profile... or TailoredProfile), the objname is the name of the resource (`.metadata.name` in kubernetes)